core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1436,9 +1436,9 @@ private String toStringWithChildren(String req, String res, int numChildren) {
     }
 
     private static StringBuilder toStringWithoutChildren(StringBuilder buf, String req, String res) {
-        return buf.append("{req=")
+        return buf.append('{')
                   .append(req)
-                  .append(", res=")
+                  .append(", ")
                   .append(res)
                   .append('}');
     }
```

Comment:
Wouldn't this make it a little bit harder to read, although it's already fairly hard to read? I'd prefer this to be reverted.

Reply:
As we discussed, I didn't revert this format.
Previous:
```
{req={...}, res={...}}"
```
Now:
```
{Request: {...}, Response: {...}}"
```